src/nxt_router.h

```diff
@@ -54,6 +54,8 @@ typedef struct {
 
     nxt_router_access_log_t  *access_log;
     nxt_tstr_t               *log_format;
+    nxt_tstr_t               *log_enable;
+    uint8_t                  log_negative;  /* 1 bit */
```

Comment:
Ah, hello my old friend, if only these comments had said bool instead, it would have saved me so much time and confusion...

But onto the real point...

What kinds of things does log_enable hold? Having had a few reads over the code, I'm currently thinking it's more like _log_condition_ or _log_expression_

Reply:
Yes, new naming is better, thanks.